tests/e2e/test-register-ingest.py

```diff
@@ -0,0 +1,158 @@
+import os
```

Comment:
Where is the ingestion (name of the file)?

Reply:
Renamed.